dev/maven-plugin/src/main/maven/plugin.xml

```diff
@@ -409,6 +409,15 @@
                 <parameter>
                     <name>cassandraEnabled</name>
                 </parameter>
+                <parameter>
+                    <name>kafkaPort</name>
+                </parameter>
+                <parameter>
+                    <name>kafkaAddress</name>
+                </parameter>
```

Comment:
Or even `devModeKafkaAddresses` to make it clear this value is for a single address _and_ only used on DevMode.